server/sonar-web/src/main/js/app/components/nav/app.js

```diff
@@ -24,13 +24,13 @@ import ReactDOM from 'react-dom';
 import GlobalNav from './global/global-nav';
 import ComponentNav from './component/component-nav';
 import SettingsNav from './settings/settings-nav';
-import { getGlobalNavigation, getComponentNavigation, getSettingsNavigation } from '../../api/nav';
+import { getGlobalNavigation, getComponentNavigation, getSettingsNavigation } from '../../../api/nav';
 
 export default class App {
   start () {
     const options = window.sonarqube;
 
-    require('../../components/workspace/main');
+    require('../../../components/workspace/main');
 
     return new Promise((resolve) => {
```

Comment:
Hi sorry to take over this but I just wanted to quickly ask this... I just noticed some env vars and I just want to make sure we're not doing anything wrong before I merge the logic for PR support on Travis on https://github.com/apache/maven/pull/97

https://github.com/axel3rd/files-archiver/pull/5#issuecomment-261673966